src/rust/src/lazy/dsl.rs

```diff
@@ -2210,6 +2205,61 @@ impl Expr {
     pub fn new_last() -> Expr {
         dsl::last().into()
     }
+
+    pub fn cov(a: Robj, b: Robj) -> RResult<Expr> {
+        Ok(pl::cov(robj_to!(PLExprCol, a)?, robj_to!(PLExprCol, b)?).into())
+    }
+
+    pub fn rolling_cov(
+        a: &Expr,
+        b: &Expr,
+        window_size: Robj,
+        min_periods: Robj,
+        ddof: Robj,
+    ) -> RResult<Self> {
+        Ok(pl::rolling_cov(
+            a.0.clone(),
+            b.0.clone(),
+            new_rolling_cov_options(window_size, min_periods, ddof)?,
+        )
+        .into())
+    }
+
+    pub fn corr(
+        a: &Expr,
```

Comment:
avoid use of `&Expr` (see comment) or any other arg type than `Robj` it is still around in code base. But every PR should chip away at old conversions.

Reply:
updated